### ISSUE TITLE
[JOSS Review] Community guidelines

### ISSUE THREAD

**Issue by V-Agudetse** [ptaconet/modisfast#8]
Hello @ptaconet, 

I am opening this issue in the context of the review of the modisfast publication in JOSS (openjournals/joss-reviews#7343) (really nice package with great usage examples, btw :) ). 

I have a couple of notes related to the community guidelines after reviewing [CONTRIBUTING.md](https://github.com/ptaconet/modisfast/blob/master/CONTRIBUTING.md):

Blocking: 
- Some of the links in the document (as well as in other documents, such as README.md) that point to files within the modisfast repository are broken because they point to a non-existing "main" branch instead of the current master branch. This interferes with the clarity of the guidelines so the links should be fixed before the "community guidelines" box can be checked.

Non-blocking: 
- This is not part of the JOSS review requirements, but the contribution guidelines state that code contributions should follow the Tidyverse style guide, but the style of a large part of the code does not follow Tidyverse conventions, especially in regards to line length and syntax issues such as spacing. I can recommend the styler package as a tool to quickly format the code to be in accordance with Tidyverse conventions.

Cheers,

Victòria A.

**Comment by ptaconet** [comment]
Hi V-Agudetse, thanks for your feedback :)
Commit https://github.com/ptaconet/modisfast/commit/8c438631c41d5270036efde385ec9bc78d1035ec fixes the "Blocking" part of the issue.
I am now working on the Non-blocking part

**Comment by ptaconet** [comment]
Commit https://github.com/ptaconet/modisfast/commit/b15397d275a813abf77e8702c03927996d8fe159 fixes part of Non-blocking suggestions. As suggested, I have ran the repository through the package 'styler' (I did not know this package, thanks for the mention!). I have made the requested changes, except for the line length limit (I must say it is very difficult to respect the 80 characters line length...). In order to take into account this limitation, in the Contributing.md file I have changed the sentence "We use the Tidyverse style guide for writing R code" to "We use as much as possible the Tidyverse style guide for writing R code - except for line length".
